script/create-dist.py

```diff
@@ -85,6 +85,7 @@ def main():
   copy_chrome_binary('chromedriver')
   copy_chrome_binary('mksnapshot')
   copy_license()
+  copy_api_json()
```

Comment:
Semantically this doesn't read very well.

Perhaps `copy_generated_json_api()`?

To be honest I can't even come up with a nice reading function name 😢